packages/css/src/image/image.scss

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-image {
+  background-repeat: no-repeat; /* [4] */
+  background-size: cover; /* [4] */
+  font-style: italic; /* [3] */
+  height: auto; /* [1] */
+  max-width: 100%; /* [1] */
+  shape-margin: 1rem; /* [5] */
+  vertical-align: middle; /* [2] */
+}
+
+// [1] Allow for fluid image sizing while maintaining aspect ratio governed by width/height attributes
+// [2] Remove ‘phantom’ whitespace
+// [3] Italicise alt text to visually offset it from surrounding copy
+// [4] Set up backgrounds for optional LQI
+// [5] Set up margin for optional `shape-outside`
```

Comment:
Shouldn't you set this together with `shape-outside` when you use it? Feels a bit unnecessary to set it as default.

Reply:
Yes, maybe. I’ll remove it for now, together with the LQI one.